openedx/features/course_experience/views/course_home.py

```diff
@@ -240,5 +241,21 @@ def render_to_fragment(self, request, course_id=None, **kwargs):  # lint-amnesty
             'has_discount': has_discount,
             'show_search': show_search,
         }
-        html = render_to_string('course_experience/course-home-fragment.html', context)
+
+        course_home_template = 'course_experience/course-home-fragment.html'
+        try:
+            # .. filter_implemented_name: CourseHomeRenderStarted
+            # .. filter_type: org.openedx.learning.course_home.render.started.v1
+            context, course_home_template = CourseHomeRenderStarted.run_filter(
+                context=context, template_name=course_home_template,
+            )
+        except CourseHomeRenderStarted.RenderAlternativeCourseHome as exc:
+            html = render_to_string(exc.course_home_template, exc.template_context)
```

Comment:
We could also call it course_home_fragment_template, but that might be threading too thin.